### DISCUSSION TITLE
Is there a way to restart your image count numbers for the ones you save?

### DISCUSSION THREAD

**Discussion by MartyDog1** [lllyasviel/stable-diffusion-webui-forge#2379] [General]
Like, if you hit the disk icon under the image, and save it as "00596-(seed)" is there a way to reset that number back to zero?

**Comment by Haoming02** [comment]
Isn't the number counting up from the existing files? If you want it to start back at zero, you would need to delete the old images in the `outputs` folder.

**Reply by MartyDog1** [reply]
I guess I should pay more attention to things. I've been hitting the disk icon, and then saving them in a different directory. Somehow never realized they were being saved in the log/images directory.
